src/shadertools/modules/diffuse/diffuse.js

```diff
@@ -0,0 +1,58 @@
+// Minimal texture (diffuse map) support
```

Comment:
I am open to suggestions. I think the initial idea was to support a texture, which I usually see referred to as a diffuse map.